cumulusci/core/config/OrgConfig.py

```diff
@@ -178,3 +180,27 @@ def get_community_info(self, community_name, force_refresh=False):
             )
 
         return self._community_info_cache[community_name]
+
+    @property
+    def installed_packages(self):
+        if not self._installed_packages:
+            response = self.salesforce_client.restful(
+                "tooling/query/?q=SELECT SubscriberPackage.NamespacePrefix, SubscriberPackageVersion.MajorVersion, "
+                "SubscriberPackageVersion.MinorVersion, SubscriberPackageVersion.PatchVersion "
+                "FROM InstalledSubscriberPackage"
```

Comment:
What about beta package versions? (IsBeta = true)

Reply:
Betas are treated as if they are the upcoming major version, i.e., 1.1.0 Beta 1 is treated as 1.1.0. I think that this will produce the correct behavior where a comparison is being performed. Automation should not be attempting to test for equality to a specific beta.

Reply:
That makes sense for the preflight check use case, but it seems like there may be other use cases where it's important to be able to tell if it's a beta or not. If we synthesized a version like the ones used on platform (i.e. 1.1.0 Beta 1 -- does that correspond to the build number?) and used LooseVersion, I think it would still work okay for the comparisons:

    >>> LooseVersion("3.175 Beta 1") >= "3.175"
    True

Reply:
I don't think that outcome is quite right, though. 3.175 Beta 1 should be <= 3.175, while 3.176 Beta 1 > 3.175 and 3.175 Beta N.

`StrictVersion` supports a `bN` suffix on the version number that seems to exhibit the right sorting behavior, e.g.,

`StrictVersion("1.10b1") < StrictVersion("1.10")` => `True`
`StrictVersion("1.10b1") < StrictVersion("1.10b2")` => `True`

The `BuildNumber` field _mostly_ corresponds to the Beta number; it's the count of builds against that major release. So 1.8 Beta 9 has BuildNumber 9, and 1.8 has BuildNumber 10. That should suffice to generate real, comparable version numbers for betas.

Reply:
I adopted the `b` model for betas. See what you think in the latest version.